examples/assetmanagement/assets.py

```diff
@@ -81,17 +82,16 @@
 
 # Query assets using id.
 query_asset_request = QueryAssetsRequest(
-    ids=[created_asset_id],
+    filter=f'AssetIdentifier = "{created_asset_id}"',
     skip=0,
     take=1,
     descending=False,
-    calibratable_only=False,
     return_count=False,
 )  # type: ignore
```

Comment:
You should be able to remove the `# type: ignore` with this change.
```suggestion
)
```